### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Dimoshua**
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,14 +4,224 @@
...
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://github.com/Dimoshua/layout_html-form.git"
+      method="post"
+      oninput="result.value=myrange.value"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label for="surname" class="block">Surname:
```
Comment (SanyaBratashchuk) [inline src/index.html:24]:
```suggestion
        <label for="surname" class="block">
          Surname:
```

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,224 @@
...
+          <input
+            type="text"
+            id="surname"
+            name="Personal_information"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="name" class="block">Name:
```
Comment (SanyaBratashchuk) [inline src/index.html:34]:
```suggestion
        <label for="name" class="block">
          Name:
```
Fix it everywhere

File: src/index.html [hunk 3]
```diff
@@ -4,14 +4,224 @@
...
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="block">Do you love cats?
```
Comment (SanyaBratashchuk) [inline src/index.html:105]: It shouldn't be a label because when I click on the question `yes` become selected

File: src/style.css [hunk 4]
```diff
@@ -1 +1,10 @@
 /* styles go here */
+
+.block {
```
Comment (SanyaBratashchuk) [inline src/style.css:3]: add this class for each fieldWrapper and use `:last-child` pseudoclass

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Dimoshua) [reply src/index.html:24]: It's done

File: src/index.html (hunk 2 above)
Comment (Dimoshua) [reply src/index.html:34]: It's done

File: src/index.html (hunk 3 above)
Comment (Dimoshua) [reply src/index.html:105]: It's done

File: src/style.css (hunk 4 above)
Comment (Dimoshua) [reply src/style.css:3]: It's done

**Pull request closed without merging**